src/renderer/renderScene.ts

```diff
@@ -116,31 +89,26 @@ export function renderScene(
     ),
   );
 
-  applyZoom(context);
```

Comment:
This was needlessly applying and resetting around each section. We only need to apply and restore once